### PR TITLE
ingested taln 2021

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

File: data/xml/2021.jeptalnrecital.xml [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+      <url hash="1518edbc">2021.jeptalnrecital-taln.5</url>
+      <language>fra</language>
+    </paper>
+    <paper id="6">
+      <title>Contribution d’informations syntaxiques aux capacités de généralisation compositionelle des modèles seq2seq convolutifs (Assessing the Contribution of Syntactic Information for Compositional Generalization of seq2seq Convolutional Networks)</title>
+      <author><first>Diana</first><last>Nicoleta Popa</last></author>
```
Comment (danielgildea) [inline data/xml/2021.jeptalnrecital.xml:72]: last name Popa

File: data/xml/2021.jeptalnrecital.xml [hunk 2]
```diff
@@ -0,0 +1,626 @@
...
+      <author><first>William</first><last>N. Havard</last></author>
```
Comment (danielgildea) [inline data/xml/2021.jeptalnrecital.xml:73]: last name Havard

File: data/xml/2021.jeptalnrecital.xml [hunk 3]
```diff
@@ -0,0 +1,626 @@
...
+      <abstract>Nous présentons un résumé en français et un résumé en anglais de l’article (Cardon &amp; Grabar, 2020), publié dans les actes de la conférence 28th International Conference on Computational Linguistics (COLING 2020).</abstract>
+      <url hash="c0261cbb">2021.jeptalnrecital-taln.24</url>
+      <language>fra</language>
+    </paper>
+    <paper id="25">
+      <title>Stratégie Multitâche pour la Classification Multiclasse (A Multitask Strategy for Multiclass Classification)</title>
+      <author><first>Houssam</first><last>Akhmouch</last></author>
+      <author><first>Hamza</first><last>Bouanani</last></author>
+      <author><first>Gaël</first><last>Dias</last></author>
+      <author><first>Jose</first><last>G Moreno</last></author>
```
Comment (danielgildea) [inline data/xml/2021.jeptalnrecital.xml:265]: last name Moreno
first name Jose G.

File: data/xml/2021.jeptalnrecital.xml (hunk 1 above)
Comment (xinru1414) [reply data/xml/2021.jeptalnrecital.xml:72]: @danielgildea Thanks!
Should I keep `Nicoleta` in the last name for first name?

File: data/xml/2021.jeptalnrecital.xml (hunk 2 above)
Comment (xinru1414) [reply data/xml/2021.jeptalnrecital.xml:73]: @danielgildea Thanks!

File: data/xml/2021.jeptalnrecital.xml (hunk 3 above)
Comment (xinru1414) [reply data/xml/2021.jeptalnrecital.xml:265]: @danielgildea thanks!

File: data/xml/2021.jeptalnrecital.xml (hunk 1 above)
Comment (danielgildea) [reply data/xml/2021.jeptalnrecital.xml:72]: I think the first name is "Diana Nicoleta"

Comment (xinru1414) [reply data/xml/2021.jeptalnrecital.xml:72]: NVM I figured it out

**Review by mjpost — APPROVED** [review]
LGTM.

**Comment by mjpost** [comment]
We’ve had some issues filed, so let’s hold off on merging this till they’re resolved.

**Pull request merged**